app/assets/javascripts/surveys.js

```diff
@@ -6,51 +6,55 @@
 
   var takeSurveyLink = function (text, className) {
     className = className ? 'class="' + className + '"' : ''
-    return '<a ' + className + ' href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">' + text + '</a>'
+    return '<a ' + className + ' href="{{surveyUrl}}" id="take-survey" target="_blank" rel="noopener noreferrer">' + text + '</a>'
   }
 
   var templateBase = function (children) {
     return (
       '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
       '  <div class="survey-wrapper">' +
       '    <a class="survey-close-button" href="#user-survey-cancel" aria-labelledby="survey-title user-survey-cancel" id="user-survey-cancel" role="button">Close</a>' +
-      '    <h2 class="survey-title" id="survey-title">Tell us what you think of GOV.UK</h2>' +
```

Comment:
We can leave the title in here can't we?  Or do we only apply template params before we pass things into `templateBase`?